samples/matter/common/src/thread_util.cpp

```diff
@@ -11,8 +11,21 @@
 #include <platform/CHIPDeviceLayer.h>
 #include <platform/internal/DeviceNetworkInfo.h>
 
+#ifdef CONFIG_MPSL_FEM
+#include <openthread/platform/radio.h>
+#include <platform/OpenThread/OpenThreadUtils.h>
+#endif
+
 #include <zephyr/zephyr.h>
 
+#ifdef CONFIG_MPSL_FEM
+#ifndef CONFIG_FEM_802_15_4_DEFAULT_TX_POWER
+#define CONFIG_FEM_802_15_4_DEFAULT_TX_POWER 20
```

Comment:
I think that setting `CONFIG_` defines like that can be misleading because for most developers `CONFIG_` constants come from Kconfig only. IMHO if an app does not define such a Kconfig it shouldn't call `SetDefaultThreadOutputPower` at all. You can check that at compile time by replacing `#ifdef CONFIG_MPSL_FEM` with `#if CONFIG_FEM_802_15_4_DEFAULT_TX_POWER` checks.